solutions/usaco-380.mdx

```diff
@@ -131,5 +131,118 @@ public class CrossCountrySkiing {
 
 </JavaSection>
 
+<CPPSection>
+
+```cpp
+// @author: ascyrax
+
+#include <bits/stdc++.h>
+
+using namespace std;
+
+#define endl "\n"
+
+#define pb push_back
+
+int r, c;
+vector<vector<int>> grid;     //grid stores the elevation
+vector<vector<bool>> visited; //this will be true if the corresponding location of the grid has been visited atleast once
```

Comment:
same here

Reply:
see prev comment

Reply:
exceeds 80 chars

Reply:
@nwatx I didn't get this "exceeds 80 chars" deal properly. Does it mean that a single row should not contain more than 80 chars limit.

Reply:
yeah

Reply:
yep!